src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -6395,6 +6415,13 @@ QString ViewProviderSketch::appendRedundantMsg(const std::vector<int> &redundant
                         redundant);
 }
 
+QString ViewProviderSketch::appendPartiallyRedundantMsg(const std::vector<int> &partiallyredundant)

```

Comment:
I know the code cut is soon, so i will NOT say these statics really are begging to be moved to an independent cpp/h pair.